test/test_mkldnn_verbose.py

```diff
@@ -0,0 +1,33 @@
+# Owner(s): ["module: unknown"]
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+import os
+import subprocess
+
+class TestMKLDNNVerbose(TestCase):
+    def test_verbose_on(self):
+        num = 0
+        loc = os.path.dirname(os.path.abspath(__file__))
+        with subprocess.Popen('python -u {}/mkldnn_verbose.py --verbose-level=1'.format(loc), shell=True,
+                              stdout=subprocess.PIPE, stderr=subprocess.STDOUT) as p:
+            for line in p.stdout.readlines():
+                line = str(line, 'utf-8').strip()
+                if line.startswith("onednn_verbose"):
+                    num = num + 1
+                elif line == 'Failed to set MKLDNN into verbose mode. Please consider to disable this verbose scope.':
+                    return
+        assert num > 0, 'oneDNN verbose messages not found.'
+
+    def test_verbose_off(self):
+        num = 0
+        loc = os.path.dirname(os.path.abspath(__file__))
+        with subprocess.Popen('python -u {}/mkldnn_verbose.py --verbose-level=0'.format(loc), shell=True,
+                              stdout=subprocess.PIPE, stderr=subprocess.STDOUT) as p:
+            for line in p.stdout.readlines():
+                line = str(line, 'utf-8').strip()
+                if line.startswith("onednn_verbose"):
+                    num = num + 1
+        assert num == 0, 'unexpected oneDNN verbose messages found.'
```

Comment:
Never use asserts in unittests, but rather family of [`unittest.assert` method](https://docs.python.org/3/library/unittest.html#assert-methods) (as language level assert can be suppressed)
```suggestion
        self.assertEqual(num,  0, 'unexpected oneDNN verbose messages found.')
```